src/efs_parser/BaseFile.py

```diff
@@ -219,12 +219,19 @@ def bin_rows(self, data, chr, start, end, columns=None, metadata=None, bins = 40
             temps = data[(data.index.left <= index.right) & (data.index.right > index.left)]
             if len(temps) > 0:
                 for col in columns:
-                    row[col] = float(np.mean(temps[col].values))
+                    #row[col] = float(np.mean(temps[col].values))
+                    row[col] = np.mean(self.change_to_float(temps[col].values))
```

Comment:
@hanyelgaml-roche 

use the pandas astype method instead - https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.astype.html#pandas-dataframe-astype